lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -1,14 +1,28 @@
 ;(function (define) {
     'use strict';
 
-    define(['backbone', 'underscore', 'text!teams/templates/teams-tab.underscore'],
-        function (Backbone, _, teamsTabTemplate) {
-            var TeamTabView = Backbone.View.extend({
-                render: function() {
-                    this.$el.html(_.template(teamsTabTemplate, {}));
-                }
-            });
+    define(['backbone',
+            'underscore',
+            'gettext',
+            'js/components/header/views/header',
+            'js/components/header/models/header',
+            'text!teams/templates/teams-tab.underscore'],
+           function (Backbone, _, gettext, HeaderView, HeaderModel, teamsTabTemplate) {
+               var headerModel = new HeaderModel({
```

Comment:
Nit: I'm curious about the way you've structured this - if, in `TeamsTabView`-land, we ever need a reference back to the header view or its model, we won't have one. I'm not saying we need to do this right now, but it feels to me that the `TeamsTabView` owns the various views it wraps, and hence should have pointers back to them.